examples/janus/README.rst

```diff
@@ -32,3 +32,9 @@ If you want to play a media file instead of sending green video frames, run:
 .. code-block:: console
 
    $ python janus.py --play-from video.mp4 http://localhost:8088/janus
+
+If you want to play a media without decode frames (just copying video container), run:
+
+.. code-block:: console
+
+   $ python janus.py --no-decode --play-from rtsp://user@password@ip:port/stream http://localhost:8088/janus
```

Comment:
I'm not sure I'm comfortable with this example, can `--no-decode` even work without forcing the video codec?

Reply:
I believe so.

Reply:
After some though I think we should focus all our efforts on making one example absolutely right, and I think that example should be the "webcam" one as it is simpler to understand and to use (you don't need a janus instance). If you're OK with this I would suggest rolling back all changes to the `janus` example and defer this for a future PR.

Reply:
Ok. I tested Janus today (with mozilla and chrome browsers) with this branch and it works. So, could you test webcam example to us? Others reviews tested at the past and it was working.

Feel free to help create a new PR with changes in janus example and I help you to test (I have an environment configured here).

Reply:
The only way it can "just work" is if Janus doesn't support vpx. Depending on this doesn't seem a solid assumption.

In any case my point still stands: there is some fine tuning to be done to the examples so users don't shoot themselves in the foot (and end up opening tickets I will have to deal with). I'd rather do it in just one place until we've got it right.

Reply:
Sorry @jlaine. I checked Janus Server and my video rooms  are configured to use only H264. Because of that I need to adjust this (I'll create a new PR with this).

About tickets related with this, feel free to mark me in issues to resolve related problem (I said that I help you before).